control_panel_api/services.py

```diff
@@ -0,0 +1,107 @@
+from django.conf import settings
+
+from . import aws
+
+READ_WRITE = 'readwrite'
+READ_ONLY = 'readonly'
+
+
+def _bucket_name(name):
+    """Prefix the bucket name with environment e.g. dev-james"""
+    return "{}-{}".format(settings.ENV, name)
+
+
+def _policy_name(bucket_name, readwrite=False):
+    """Prefix the policy name with bucket name, postfix with access level e.g. dev-james-readwrite"""
+    return "{}-{}".format(bucket_name, READ_WRITE if readwrite else READ_ONLY)
+
+
+def _policy_arn(bucket_name, readwrite=False):
+    return "{}:policy/{}".format(settings.IAM_ARN_BASE, _policy_name(bucket_name, readwrite))
+
+
+def _bucket_arn(bucket_name):
+    return "arn:aws:s3:::{}".format(bucket_name)
+
+
+def get_policy_document(bucket_name, readwrite):
```

Comment:
It may be the case of make this function "private" as well (=> `_get_policy_document()`) as I don't think the client code would really use it directly.